### DISCUSSION TITLE
getting null for a collections snapshots

### DISCUSSION THREAD

**Discussion by emadbaqeri** [invertase/react-native-firebase#6253] [Q&A]
I'm using firebase, this is the implementation code that used to work until I have changed my firebase account and reconfigured it.
In the snippet below I'm trying to connect to firestore and get snapshots from a collection which it returns null.
```javascript
  React.useEffect(() => {
    const subscribe = firestore()
      .collection("messaging")
      .onSnapshot(s => console.log(s));

    return () => subscribe();
  }, []);
```
The screen shot below confirms that I have some documents in the collection that the firestore could subscribe for.

<img width="325" alt="Screen Shot 2022-05-16 at 1 16 07 PM" src="https://user-images.githubusercontent.com/50639499/168554462-a9c26101-55c5-40ff-bc07-0e03b7d25962.png">

Any Ideas would be appreciated ;)

**Comment by mikehardy** [comment]
Unfortunately I'm not sure what we can really do here, this is the phrase that really sticks out to me:

> I'm using firebase, this is the implementation code that used to work until I have changed my firebase account and reconfigured it.

Specifically

> until I have changed my firebase account and reconfigured it.

Carefully examine the differences between your accounts and their configurations, and how those are applied in the app. It sounds like it will be related to that, as the firestore module here is working fine
